lib/workers/bin/worker.rb

```diff
@@ -1,2 +1,10 @@
 type = ARGV.shift
+begin
+  # Remove the ::Runner class which is in all of them.
+  worker_name = type.gsub(/::Runner$/, '')
+  # Will set the process' name to eg. 'EVM Worker: MiqPriorityWorker'
+  # in ps aux, htop, ...
+  Process.setproctitle("EVM Worker: #{worker_name}")
```

Comment:
I like that too, that's what I was doing locally for the forking workers PR that's still WIP:
https://github.com/ManageIQ/manageiq/pull/3593/files#diff-a7ce0fdab9c89206c1d9565fd7eddc31R468

I'd like @akrzos to give his input since he's also parsing top/ps/etc. for his work so he probably has some ideas.

My one concern with ManageIQ Worker is the amount of wasted space for the prefix.

See below where the smem utility truncates the title to fit in it's display:
![image](https://cloud.githubusercontent.com/assets/19339/12185319/48dbaf54-b568-11e5-9f6d-9fe1daab9481.png)

The current display in my WIP PR prepends ManageIQ Worker to the existing one that includes the id and other information... this is only visible in ps and utilities that don't truncate it:

![image](https://cloud.githubusercontent.com/assets/19339/12185368/9465b5e6-b568-11e5-8e66-123ba5078804.png)